styles.css

```diff
@@ -0,0 +1,36 @@
+#main{
+  background-color: #eee;
+  text-align: center;
+  padding: 2rem 1rem 0.5rem 1rem;
+  margin: 1.5rem 0;
+}
+
+header{
+  margin: 2rem 1rem 1rem 1rem;
+}
+
+#img-div{
+  background-color: white;
+  padding: 0.5rem 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+img {
+  max-width: 100%;
+}
+#tribute-info{
```

Comment:
Leaving a single white space between the selector and open curly brace would make your codebase clean and more readble. Doing so will leave no errors with the linters.